packages/material-ui-types/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@material-ui/types",
+  "version": "1.0.0",
+  "private": false,
+  "author": "Material-UI Team",
+  "description": "Material-UI Types - Utility types for Material-UI.",
+  "types": "./index.d.ts",
+  "keywords": [
+    "react",
+    "react-component",
+    "material design",
+    "material-ui",
+    "types"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mui-org/material-ui.git"
+  },
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/mui-org/material-ui/issues"
+  },
+  "homepage": "https://github.com/mui-org/material-ui/tree/next/packages/material-ui-types",
+  "scripts": {
+    "release": "npm publish --tag latest",
```

Comment:
Does this work for publishing? @oliviertassinari

Reply:
Publishing them from `@material-ui/utils` wouldn't work or it's not appropriate?

Reply:
I don't think it's appropriate since it's really only types. We can do it but it seems weird to pull in a runtime library for compile time only. 

Especially since `@material-ui/pickers` relies on it.

Reply:
It looks good.